src/test/java/nl/tudelft/ewi/devhub/webtests/AssignmentTest.java

```diff
@@ -1,16 +1,20 @@
 package nl.tudelft.ewi.devhub.webtests;
 
-import static org.junit.Assert.assertEquals;
-
-import java.util.List;
-
 import nl.tudelft.ewi.devhub.webtests.utils.WebTest;
+import nl.tudelft.ewi.devhub.webtests.views.AssignmentView;
 import nl.tudelft.ewi.devhub.webtests.views.AssignmentsView;
-
 import org.junit.Test;
 
+import java.text.ParseException;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
 public class AssignmentTest extends WebTest {
 
+	private static final String ASSISTANT_USERNAME = "assistant1";
+	private static final String ASSISTANT_PASSWORD = "assistant1";
+
```

Comment:
You can add `@Inject CourseEditions courseEditions` and `@Inject Assignments assignments` here and then do 
`courseEditions.find(1).get()` and `assignments.find(courseEdition, 1)` to retrieve the actual `CourseEdition` and `Assignment` instances, so you would not have to hard code strings as `Student Two`, `Product Vision` and `Submitted` (just as a mere example on how you can get more control over the various parts in Devhub in your test)